.github/workflows/publish_shacl.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish SHACL shape for current version
+
+# on:
+#   push:
+#     branches:
+#       - master
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install .[all]
+    - name: Publish shape
+      run: |
+        git clone https://${{secrets.SHACL_ACCESS_TOKEN}}@github.com/Panaetius/SHACL_test_repo.git shacl_publish
```

Comment:
which user does the `SHACL_ACCESS_TOKEN` belong to? We should use the renku bot user for this

Reply:
Right now it's my access token as it deploys to a testrepo under my name. THis has to be changed before merging (as mentioned above), not sure I have the rights to do that though.